Scan/Devices/MatrixARM/matrix_scan.c

```diff
@@ -19,6 +19,12 @@
  * THE SOFTWARE.
  */
 
+
+/*
+ * NOTE! This has been depricated in favor of the MatrixARMPeriodic implementation.
```

Comment:
deprecated

Reply:
I was looking through all the modules to try finding mcu specific register code. I almost started ifdefing'ing this file too until I realized it was deprecated when looking through other files. I figured I might as well give the extra heads up to anyone else who stumbles into the file. I can remove the message if you want though (it might be better for a different commit/time). It's no big deal either way.